### ISSUE TITLE
책에서 궁금사항 있어 질문드립니다.

### ISSUE THREAD

**Issue by gfzgfz** [rlcode/reinforcement-learning-kr#9]
page.53쪽에서 

오른쪽으로 행동을 취할 경우 1의 보상을 받는다고 적혀있는데,

아래->오른쪽, 오른쪽->아래 의 행동을 취해야 1의 보상을 받는게 아닌지요.

잘이해가 가지 않아 질문드립니다.

그리고, 상,하,좌,우 에대한 가치함수는 임의로 정한 것이지요?

답변기다리겠습니다. 감사합니다.

**Comment by kekmodel** [comment]
저도 처음에 헷갈렸던 부분인데 임의로 정한 것 같습니다.

**Comment by wooridle** [comment]
네 @kekmodel 님의 말씀이 맞습니다. 그림 2.17에 한정지어 "오른쪽으로 행동했을 때 1이라는 보상을 받는다"를 의미합니다.

**Comment by YunanJeong** [comment]
좋은 책 써주셔서 잘보고 있습니다.  저도 이 그림에서 헷갈렸었는데요.
그럼 코멘트에 이어서,
해당 그림에서 별표는  '화살표가 보상을 가지고 있다'라는 의미만 있는 것이고 목적지를 나타낸 것은 아니라고 봐도 되는건가요?

**Comment by dnddnjs** [comment]
@YunanJeong 네 맞아요. 별표가 위치한 곳이 다른 상태라서 좀 헷갈리는 것 같아요. 개정판에서는 그림을 수정할 예정이에요!
